python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -647,6 +647,54 @@ def TensorsConcatPromoteDTypeModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ConcatStaticModule(torch.nn.Module):
```

Comment:
can you change this class name to "TensorConcatStaticModule"? So it's aligned with other test names. Thanks.